site/en/gemini-api/docs/semantic_retrieval.ipynb

```diff
@@ -70,7 +70,7 @@
         "\n",
```

Comment:
<p>Same here I guess - do we want to talk about "the Generative Language API" ? Understood if it's out of scope for this change, just made me wonder.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/446/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Done. Gemini API.</p>